### PR TITLE
ref(onboarding): Make create sample event wait for event

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Kind of hard to test in dev, but this should work.\

This component could uses some tests, and the logic should probably be pulled into an action creator.

### CODE REVIEW

**Comment by fpacifici** [comment]
If you need to test in dev a delay in event ingestion you can do this:
- set SENTRY_EVENTSTREAM = 'sentry.eventstream.kafka.KafkaEventStream' in your settings
- start the devservices (it will start Kafka) skipping snuba
- run snuba separately but do not start the consumer. The event will be there on Kafka waiting. 
- when you decide to start the consumer the event will be processed and stored in snuba.

**Review by markstory — COMMENTED** [review]
Looks pretty good to me.

**Review by markstory — APPROVED** [review]

**Comment by EvanPurkhiser** [comment]
I'll add a test for the whole thing.

**Comment by EvanPurkhiser** [comment]
Updated with tests.

File: src/sentry/static/sentry/app/views/onboarding/createSampleEventButton.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    trackAdhocEvent({
+      eventKey: 'sample_event.button_viewed',
+      org_id: organization.id,
+      project_id: project.id,
+      source,
+    });
+  }
+
+  createSampleGroup = async () => {
+    // TODO(dena): swap out for action creator
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/onboarding/createSampleEventButton.tsx]: I don't think Dena is going to come back to this :cry:

**Review by markstory — APPROVED** [review]

**Review by billyvg — APPROVED** [review]

**Pull request merged**